python/dataproc_templates/util/template_constants.py

```diff
@@ -232,3 +232,9 @@
 SNOWFLAKE_TO_GCS_OUTPUT_FORMAT = "snowflake.to.gcs.output.format"
 SNOWFLAKE_TO_GCS_PARTITION_COLUMN = "snowflake.to.gcs.partition.column"
 
+# Hive Spark DDL to BQ
+HIVE_DDL_EXTRACTOR_OUTPUT_DATASET = "hive.ddl.extractor.output.dataset"
+HIVE_DDL_EXTRACTOR_OUTPUT_TABLE = "hive.ddl.extractor.output.table"
+HIVE_DDL_EXTRACTOR_INPUT_DATABASE = "hive.ddl.extractor.input.database"
+HIVE_DDL_EXTRACTOR_OUTPUT_BUCKET = "hive.ddl.extractor.output.bucket"
+HIVE_DDL_EXTRACTOR_OUTPUT_PATH = "SparkDDL/DDLs"
```

Comment:
This constant should be removed since you would you expect the paths from the user as I stated in my other comment